### PR TITLE
added addtocalendar button to single event view

### PR DESCRIPTION
**Pull request by thiagoferreiraw**

### CODE REVIEW

**Comment by kahenvay** [comment]
Added add to calendar button to single event / event details view (show_event.html).

Screenshot 2019-05-03 at 15 08 34

**Comment by thiagoferreiraw** [comment]
It didn't work for me buddy. Here is the error report the `AddEvent` created:

image

https://www.addevent.com/create/?service=appleical&dstart=7%20de%20Mai%20de%202019%20%C3%A0s%2000%3A00&dend=7%20de%20Mai%20de%202019%20%C3%A0s%2000%3A00&dsum=aaaaaaaaaa&ddesc=aaaaaaaaaaaaaaaaaaaaa&dloca=Brick%20Caf%C3%A9%20Lisboa&reference=http://localhost:8000

It's probably related to the date i10n. Perhaps we have to format before sending them?

**Comment by thiagoferreiraw** [comment]
Also, please check if it works with different timezones ;)  Thank you!

**Comment by kahenvay** [comment]
Ah, nice catch. Sorry I missed that. I'll solve it tomorrow

File: get_together/templates/get_together/events/show_event.html [hunk 1]
```diff
@@ -9,6 +9,14 @@
   <meta property="og:title"         content="{{event.name}}" />
   <meta property="og:description"   content="{{event.summary}}" />
   <meta property="og:image"         content="{{event.team.full_img_url}}" />
+  <!-- AddEvent script -->
+  <script type="text/javascript" async defer>
+  
+    
+    console.log('date', ('{{event.local_start_time}}'));
+    // console.log('formatDate', formatDate('{{event.local_start_time}}'));
```
Comment (pawalls-zz) [inline get_together/templates/get_together/events/show_event.html]: Please clean-up comment before submit.

File: get_together/templates/get_together/events/show_event.html [hunk 2]
```diff
@@ -104,7 +112,7 @@
                 <h2 class="team-title">{{ event.name }}{% if event.status == event.CANCELED %} ({% trans "Canceled" %}){% endif %}</h2>
             </div>
             {% else %}
-            <h2>{{ event.name }}</h2>
+            <h2> {{ event.name }}</h2>
```
Comment (pawalls-zz) [inline get_together/templates/get_together/events/show_event.html]: Stray whitespace or is this intentional?

**Review by pawalls-zz — CHANGES_REQUESTED** [review]
Looks great overall, @kahenvay! Minor whitespace/style knits.

File: get_together/templates/get_together/events/show_event.html [hunk 3]
```diff
@@ -9,6 +9,14 @@
...
   <meta property="og:description"   content="{{event.summary}}" />
   <meta property="og:image"         content="{{event.team.full_img_url}}" />
+  <!-- AddEvent script -->
+  <script type="text/javascript" async defer>
+  
+    
+    console.log('date', ('{{event.local_start_time}}'));
+    // console.log('formatDate', formatDate('{{event.local_start_time}}'));
+  </script>
+<script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js" async defer></script>
```
Comment (pawalls-zz) [inline get_together/templates/get_together/events/show_event.html:13]: Non-blocking FYI:

Ultimately, we will want to host all Javascript ourselves. Third-party JS makes our frontend less likely to work with things like uBlock/uMatrix and also opens our users to external tracking. In the long run, we may need to push emergency fixes for these as well (for security, etc).

We need to establish a pattern here and a mechanism for vendoring. I've thrown a story for this on the icebox.

File: get_together/templates/get_together/events/show_event.html (hunk 2 above)
Comment (kahenvay) [reply get_together/templates/get_together/events/show_event.html]: Stray, removed

File: get_together/templates/get_together/events/show_event.html (hunk 1 above)
Comment (kahenvay) [reply get_together/templates/get_together/events/show_event.html]: Yes sorry! I made myself a little checklist to look at before pushing now.

**Comment by thiagoferreiraw** [comment]
PR reviewed Tested! 
Thank you @kahenvay !

**Pull request merged**